### PR TITLE
Add the installing section.

### PR DESCRIPTION
**Pull request by jadenisco**

### CODE REVIEW

File: docs/setup/installing/index.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+Installing on openSUSE
+----------------------------------------
+The following are instructions on how to install VPP on openSUSE.
+
+.. toctree::
+
+   opensuse
+
+Downloading the jvpp jar
```
Comment (mdr78) [inline docs/setup/installing/index.rst]: This doesn't tell me much about what the jvpp jar is  - or why I would want to install it.

File: docs/setup/installing/jvpp.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
+.. _opensuse:
```
Comment (mdr78) [inline docs/setup/installing/jvpp.rst]: Does this belong here?

File: docs/setup/installing/ubuntu.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+.. code-block:: console
+
+  export RELEASE=".master"
+
+**3. To write the fd.io sources list execute:**
+
+.. code-block:: console
+
+  sudo rm /etc/apt/sources.list.d/99fd.io.list
+  echo "deb [trusted=yes] https://nexus.fd.io/content/repositories/fd.io$RELEASE.ubuntu.$UBUNTU.main/ ./" | sudo tee -a /etc/apt/sources.list.d/99fd.io.list
```
Comment (mdr78) [inline docs/setup/installing/ubuntu.rst]: Is some cases above you ask folks to edit files, in others you pipe stdin - should be made consistent.

File: docs/setup/installing/opensuse.rst [hunk 4]
```diff
@@ -0,0 +1,50 @@
+.. _opensuse:
+
+.. toctree::
+
+Blog post
```
Comment (mdr78) [inline docs/setup/installing/opensuse.rst]: Suggest 'See Also'

File: docs/setup/installing/ubuntu.rst [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+Install the mandatory packages
+--------------------------------
+
+.. code-block:: console
+
+  sudo apt-get update
```
Comment (mdr78) [inline docs/setup/installing/ubuntu.rst]: Will there be an repository signing warnings to be accepted etc.

File: docs/setup/installing/opensuse.rst [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+Top install VPP on openSUSE first pick the following release and execute the appropriate commands.
+
+openSUSE Tumbleweed (rolling release)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: console
+
+   sudo zypper install vpp vpp-plugins
+
+openSUSE Leap 42.3
```
Comment (mdr78) [inline docs/setup/installing/opensuse.rst]: What the relationship between the version of Leap and openSUSE version?

File: docs/setup/installing/centos.rst [hunk 7]
```diff
@@ -0,0 +1,68 @@
+.. _centos:
+
+.. toctree::
+
+The fd.io repo
```
[the remark this answers is not in the document]
Comment (jadenisco) [reply docs/setup/installing/centos.rst]: I'll add this

File: docs/setup/installing/jvpp.rst (hunk 2 above)
Comment (jadenisco) [reply docs/setup/installing/jvpp.rst]: Nope

File: docs/setup/installing/opensuse.rst (hunk 4 above)
Comment (jadenisco) [reply docs/setup/installing/opensuse.rst]: Moved to the bottom of the page and called it "For more Information"

File: docs/setup/installing/index.rst (hunk 1 above)
Comment (jadenisco) [reply docs/setup/installing/index.rst]: This should go in the API Section. jvpp.jar is the VPP api.

**Pull request merged**